src/main/java/io/nats/client/impl/Headers.java

```diff
@@ -448,6 +452,8 @@ public byte[] getSerialized() {
 	/**
 	 * @deprecated
 	 * Appends the serialized bytes to the builder.
+	 * 
+	 * @param bab the ByteArrayBuilder to append
```

Comment:
This isn't used and probably was public only for testing, it was unfortunate I released it this way, but I can't take it back. I would just make this super deprecated